src/components/Categorizer.js

```diff
@@ -17,75 +17,72 @@ class Categorizer extends React.Component {
     });
   };
 
-  updateSelectedTransactions = newSelectedTransactions => {
+  updateSelectedParties = newSelectedParties => {
     this.setState({
-      selectedTransactions: [...newSelectedTransactions]
+      selectedParties: [...newSelectedParties]
     });
   };
 
-  availableTransactions = (transactionsList, categories) => {
-    const transactionsFoundInCategories = [];
+  unCategorizedParties = (parties, categories) => {
+    const categorizedParties = [];
 
     if (categories && categories.length > 0) {
-      categories.map(category => {
-        if (category.categoryParties) {
-          category.categoryParties.forEach(parties => {
-            if (!transactionsFoundInCategories.includes(parties)) {
-              transactionsFoundInCategories.push(parties);
-            }
-          });
-        }
+      categories.forEach(category => {
```

Comment:
was the use of map fn here causing the bug of transactions && categories not showing correctly?

Reply:
No, I believe the problem was a check something like `if (!this.state.userState) { ...` in componentDidMount(), and there `userState` was always undefined and therefore it was always reset